test/test-udp-send-unreachable.c

```diff
@@ -59,12 +59,11 @@ static void close_cb(uv_handle_t* handle) {
 
 static void send_cb(uv_udp_send_t* req, int status) {
   ASSERT(req != NULL);
-  ASSERT(status == 0);
+  ASSERT(status == 0 || status == UV_ECONNREFUSED);
```

Comment:
We could expand this test with a new client that uses the new flag.

Reply:
I expanded the test, but the condition needs to stay, as there's no guarantee that the kernel will actually return `UV_ECONNREFUSED` when running the test.